src/core/main.c

```diff
@@ -582,6 +583,53 @@ static int config_parse_cpu_affinity2(
         return 0;
 }
 
+static int config_parse_numa_policy(
```

Comment:
A somewhat tangential note: would it make sense to move the `core/main.c`-specific `config_parse` functions (not only the NUMA ones, but the rest of them as well) to, for example, `shared/config-parser.*`, so a fuzzer could be written around the config-parsing machinery (mainly the separate `config_parse*` functions)? @keszybz

Reply:
Yes, it would. Do you want to do this?

Reply:
Sure, will look into that.